app/build.gradle

```diff
@@ -169,10 +171,10 @@ dependencies {
     testImplementation "com.nhaarman.mockitokotlin2:mockito-kotlin:$mockitoKotlin"
     testImplementation "junit:junit:4.12"
 
-    androidTestImplementation 'androidx.test:runner:1.1.0'
-    androidTestImplementation 'androidx.test:rules:1.1.0'
-    androidTestUtil "androidx.test:orchestrator:1.1.0"
-    androidTestImplementation "androidx.test.espresso:espresso-core:3.1.0"
+    androidTestImplementation "androidx.test:runner:$testRunner"
+    androidTestImplementation "androidx.test:rules:$testRunner"
+    androidTestUtil "androidx.test:orchestrator:$testRunner"
+    androidTestImplementation "androidx.test.espresso:espresso-core:3.1.1"
     androidTestImplementation "org.mockito:mockito-android:$mockito"
```

Comment:
for uniformity , it could be an extra property too, as well as the versions of commons-math3 & junit at lines: 168, 172